src/main/java/seedu/duke/Duke.java

```diff
@@ -0,0 +1,72 @@
+package seedu.duke;
+
+import java.io.IOException;
+import java.text.ParseException;
+
+/**
+ * Represents a Personal Assistant Chatbot that helps a person to keep track of various things.
+ *
+ * @author Evon Dong
+ * @version 1.0
+ * @since 2019-08-28
+ */
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructor for class Duke.
+     * Loads the data if datafile exists.
+     *
+     * @param filePath the path to the file where the list of tasks is stored
+     * @throws IOException On input error
+     */
+    public Duke(String filePath) throws IOException {
```

Comment:
Since `DukeException` is the base class of all exceptions thrown in Duke, is it better to catch the `IOException` and only throw `DukeException`? Similarly for `ParseException`.